pkg/devfile/image/docker_compatible.go

```diff
@@ -57,21 +69,29 @@ func (o *DockerCompatibleBackend) Build(image *devfile.ImageComponent, devfilePa
 	return nil
 }
 
-func getShellCommand(cmdName string, image *devfile.ImageComponent, devfilePath string) string {
+func getShellCommand(cmdName string, image *devfile.ImageComponent, devfilePath string) []string {
```

Comment:
A comment that briefly explains what `getShellCommand` function does would be helpful. I understand that I'm requesting comment for an internal function.

Reply:
ok